velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -138,10 +138,13 @@ TpchPlan TpchQueryBuilder::getQ1Plan() const {
   common::test::SubfieldFiltersBuilder filtersBuilder;
   // DWRF does not support Date type. Use Varchar instead.
   if (selectedRowType->findChild(shipDate)->isVarchar()) {
-    filtersBuilder.add(shipDate, common::test::lessThanOrEqual("1998-09-02"));
+    filtersBuilder.add(
+        columnAliases.at(shipDate),
```

Comment:
The use of term "column alias" here is confusing. Typically, we have names in the tables and files and aliases in the query. Here, we use "alias"  in the opposite way. It would be nice to change this to avoid confusion in the future.

Reply:
I renamed it to use `fileColumnNames`.